sbi/neural_nets/estimators/base.py

```diff
@@ -480,6 +561,565 @@ def diffusion_fn(self, input: Tensor, times: Tensor) -> Tensor:
         raise NotImplementedError("Diffusion is not implemented for this estimator.")
 
 
+class MaskedConditionalVectorFieldEstimator(MaskedConditionalEstimator, ABC):
```

Comment:
I think we could save some lines of code here if we use MixIn classes, like for NeuralInference or the ScoreEstimators i.e. we could share quite a few of the functions, but not sure.